cmd/podman/pod_create.go

```diff
@@ -0,0 +1,118 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/cmd/podman/libpodruntime"
+	"github.com/projectatomic/libpod/libpod"
+	"github.com/sirupsen/logrus"
+	"github.com/urfave/cli"
+)
+
+var podCreateDescription = "Creates a new empty pod. The pod ID is then" +
+	" printed to stdout. You can then start it at any time with the" +
+	" podman pod start <pod_id> command. The pod will be created with the" +
+	" initial state 'created'."
+
+var podCreateFlags = []cli.Flag{
+	cli.BoolFlag{
+		Name:  "cgroup-to-ctr",
+		Usage: "Tells containers in this pod to use the cgroup created for the pod",
+	},
+	cli.StringFlag{
+		Name:  "cgroup-parent",
+		Usage: "Optional parent cgroup for the pod",
+	},
+	cli.StringSliceFlag{
+		Name:  "label-file",
+		Usage: "Read in a line delimited file of labels (default [])",
+	},
+	cli.StringSliceFlag{
+		Name:  "label, l",
+		Usage: "Set metadata on pod (default [])",
+	},
+	cli.StringFlag{
+		Name:  "name, n",
+		Usage: "Assign a name to the container",
+	},
+	cli.StringFlag{
+		Name:  "podidfile",
+		Usage: "Write the pod ID to the file",
+	},
+}
+
+var podCreateCommand = cli.Command{
+	Name:                   "create",
+	Usage:                  "create but do not start a pod",
+	Description:            podCreateDescription,
+	Flags:                  podCreateFlags,
+	Action:                 podCreateCmd,
+	ArgsUsage:              "IMAGE [COMMAND [ARG...]]",
```

Comment:
`pod create` doesn't seem to take positional arguments, although perhaps it should, to allow you to override the default pause image.  For example, CRI-O has [`--pause-command` and `--pause image`][1].  I'd recommend:

    [IMAGE [COMMAND [ARG ...]]]

here.

[1]: https://github.com/kubernetes-incubator/cri-o/blame/v1.11.0/docs/crio.8.md#L103-L105